core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncer.java

```diff
@@ -61,11 +68,16 @@ public ActiveSyncer(FileSystemMaster fileSystemMaster, ActiveSyncManager syncMan
     mSyncManager = syncManager;
     mMountId = mountId;
     mMountTable = mountTable;
+    mSyncTasks = new LinkedBlockingQueue<>(32);
   }
 
   @Override
   public void heartbeat() {
     LOG.debug("start Active Syncer heartbeat");
```

Comment:
NIT: can we add the mount id, and maybe the mount path in this message?

Reply:
sure